src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -708,14 +708,14 @@ static ucs_status_t uct_ud_mlx5_iface_create_qp(uct_ib_iface_t *ib_iface,
 
     status = uct_ib_mlx5_iface_create_qp(ib_iface, qp, &attr);
     if (status != UCS_OK) {
-        goto err_destroy_qp;
+        return status;
```

Comment:
minor: "goto err" "err: return status"
so all err flow would be from once place